usr/bin/steamos-session-select

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/bash
```

Comment:
This file is still names steamos-session-select

Reply:
Yes. That is the executable Steam client is looking for. I'm hooking it up to my own and allowing for an OS to hook it up further downstream to do things like we do in ChimeraOS. This file needs to be called this way, tho because of Valve...